frontend/app/controllers/Giraffe.scala

```diff
@@ -1,146 +1,22 @@
 package controllers
 
 import com.gu.i18n._
-import com.gu.stripe.Stripe
-import configuration.Config
-import com.gu.stripe.Stripe.Serializer._
-import forms.MemberForm.supportForm
-import play.api.libs.concurrent.Execution.Implicits._
-import play.api.libs.json.{JsArray, JsString, Json}
 import play.api.mvc._
-import services.{AuthenticationService, TouchpointBackend}
-import com.netaporter.uri.dsl._
-import views.support.{TestTrait, _}
-
-import scalaz.syntax.std.option._
-import scala.concurrent.Future
-import utils.RequestCountry._
-import com.netaporter.uri.dsl._
-import com.netaporter.uri.{PathPart, Uri}
-import Redirects.getRedirectCountryCodeGiraffe
 
 object Giraffe extends Controller {
 
-  val social: Set[Social] = Set(
-    Twitter("I've just contributed to the Guardian. Join me in supporting independent journalism https://membership.theguardian.com/contribute"),
-    Facebook("https://membership.theguardian.com/contribute")
-  )
-
-
-  val chargeId = "charge_id"
+  val CampaignCodesToForward = Set("INT", "CMP", "mcopy")
```

Comment:
Doh! `"INT"` should be `"INTCMP"` - fixed with 6b8c27bd4625a08e87bfa9d725be82df98e0007d